ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -455,7 +470,7 @@ const ModuleBox = (props: {clickable: boolean, action: Function, children}) => {
 interface ModuleProps {
   profile: Profile,
   moduleName: AccessModule;
-  active: boolean;    // is this the currently-active module that the user should complete
+  active: boolean; // whether the user can interact with this module currently
```

Comment:
the comment describes the "clickable" concept rather than "active"

Reply:
(let's roll this into the general module states discussion)

Reply:
Interesting - I didn't realize that clickable was considered a "module state". From looking at the current code, my interpretation was that clickable is a UI property for the row. A module can only be clicked when active (it is always set to the same value, currently).

Two shapes of solutions I can potentially see here:
1. Track an `active` module as we did before, but separately track a `clickability` state - likely computed in a similar way to what I've done here, since this state is derived from the full list of modules
2. Track a list of active modules or clickable modules (whatever you want to call them). Consider the first in the list to be granted the "Next step" CTA. This is the approach in the current PR.

Reply:
Per slack discussion, your prior preference was for option (1) here. I updated the PR accordingly.